packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -54,8 +54,8 @@ export class PlaceholderCommon extends React.Component<PlaceholderProps> {
     rendering: PropTypes.object,
     fields: PropTypes.object,
     params: PropTypes.object,
-    missingComponentComponent: PropTypes.object,
-    errorComponent: PropTypes.object,
+    missingComponentComponent: PropTypes.func,
```

Comment:
Would you mind changing this (and `errorComponent` as well) to `PropTypes.node`? That way people can pass in class, func, string, fragment, etc... as a component.

Reply:
Sure thing!